dev/reload.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+set -xeuo pipefail
+
+# Script for making it easy to make a change locally and redeploy
+pushd ../bin
+  ./build
+popd
+
+docker tag "secrets-provider-for-k8s:dev" \
+         "${DOCKER_REGISTRY_PATH}/${APP_NAMESPACE_NAME}/secrets-provider"
+docker push "${DOCKER_REGISTRY_PATH}/${APP_NAMESPACE_NAME}/secrets-provider"
+
+echo "Running Deployment Manifest"
+wait_for_it 600 "$ENV_DIR/app-env.sh.yml | $cli_without_timeout apply -f -"
```

Comment:
i would also re-run the other files that define k8s resources so i can change them in my dev env. For example, i would like to modify the k8s secret to have different values.

Reply:
@sigalsax ^